source/jupyter/wire/message.d

```diff
@@ -87,6 +87,9 @@ struct Message {
 
         header.date = (cast(DateTime)Clock.currTime).toISOExtString;
         header.msgID = randomUUID.toString;
+	// set username to non-null string otherwise the you can get
+	// failures on jupyterlab
+	header.userName = "username";
```

Comment:
Please indent with spaces.

I have been trying to find a sensible value for this field. It makes little sense to me. I see this in the docs:

> In addition, there is a *username* field, e.g. for the process that generated the message, if applicable. This can be useful in collaborative settings where multiple users may be interacting with the same kernel simultaneously, so that frontends can label the various messages in a meaningful way.

That I understand, but what help is it if the kernel returns the same one each time? Or should it take the value from a message it is responding to, if any?

For now, the "username" string is ok, but obviously wrong of course.